website/views/options/inventory/menu.jade

```diff
@@ -20,6 +20,9 @@ ul.options-menu
   li(ng-class="{ active: $state.includes('options.inventory.seasonalshop') }")
     a(ui-sref='options.inventory.seasonalshop')
       =env.t('seasonalShop')
+  div.pull-right
```

Comment:
The div is unnecessary. Putting in a class or id without an element assumes a div.